apps/omg_watcher/mix.exs

```diff
@@ -1,62 +1,46 @@
-defmodule OMG.Watcher.Mixfile do
+defmodule OMG.Watcher.MixProject do
   use Mix.Project
 
   def project do
     [
       app: :omg_watcher,
-      version: "#{String.trim(File.read!("../../VERSION"))}",
+      version: "0.1.0",
       build_path: "../../_build",
       config_path: "../../config/config.exs",
       deps_path: "../../deps",
       lockfile: "../../mix.lock",
-      elixir: "~> 1.4",
       elixirc_paths: elixirc_paths(Mix.env()),
-      compilers: [:phoenix] ++ Mix.compilers(),
+      elixir: "~> 1.8",
       start_permanent: Mix.env() == :prod,
-      deps: deps(),
-      test_coverage: [tool: ExCoveralls]
+      deps: deps()
     ]
   end
 
+  defp elixirc_paths(:prod), do: ["lib"]
+  defp elixirc_paths(_), do: ["lib", "test/support"]
+
+  # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      mod: {OMG.Watcher.Application, []},
+      extra_applications: [:logger],
       start_phases: [{:attach_telemetry, []}],
-      extra_applications: [:logger, :runtime_tools, :telemetry]
+      mod: {OMG.Watcher.Application, []}
     ]
   end
 
-  # Specifies which paths to compile per environment.
-  defp elixirc_paths(:prod), do: ["lib"]
-  defp elixirc_paths(_), do: ["lib", "test/support"]
-
+  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:postgrex, "~> 0.14"},
-      {:ecto_sql, "~> 3.1"},
-      {:telemetry, "~> 0.4.0"},
-      {:spandex_ecto, "~> 0.6.0"},
-      # there's no apparent reason why libsecp256k1, spandex and distillery need to be included as dependencies
-      # to this umbrella application apart from mix ecto.gen.migration not working, so here they are, copied from
-      # the parent (main) mix.exs
-      {:libsecp256k1, git: "https://github.com/omisego/libsecp256k1.git", branch: "elixir-only", override: true},
-      {:spandex, "~> 2.4",
-       git: "https://github.com/omisego/spandex.git", branch: "fix_dialyzer_in_macro", override: true},
-      {:distillery, "~> 2.1", runtime: false},
```

Comment:
why is it now possible to remove things like `spandex` or `distillery`? I'd expect them to still be required by the security  watcher